packages/ui/src/fields/Upload/HasMany/index.tsx

```diff
@@ -67,7 +67,7 @@ export function UploadComponentHasMany(props: Props) {
       >
         {fileDocs.map(({ relationTo, value }, index) => {
           const id = String(value.id)
-          const url: string = value.thumbnailURL || value.url
+          const url: string = value.url
```

Comment:
This is not exactly the correct change. This will now force the browser to download a higher res image just to display in the thumbnail. I think that `RelationshipContent` should take a thumbnail src and pass that down the the thumbnail and use the `src` on the anchor tag that wraps the file name.